lib/cisco_node_utils/bgp.rb

```diff
@@ -154,6 +154,19 @@ def set_args_keys_default
       end
     end
 
+    # This is an enable-only method for enabling the 'nv overlay evpn' feature
```

Comment:
You don't need these feature methods. Please remove them. See comment on line 520.